internal/pkg/service/common/httpserver/middleware/otel.go

```diff
@@ -45,8 +46,7 @@ func OpenTelemetry(tp trace.TracerProvider, mp metric.MeterProvider, cfg Config)
 
 			// Create dropped span for filtered request, so child spans won't appear in the telemetry too.
 			if isRequestIgnored(req) {
-				ctx, span = tracer.Start(ctx, SpanName, trace.WithAttributes(attribute.Bool(attrManualDrop, true)))
-				ctx = context.WithValue(ctx, RequestSpanCtxKey, span)
+				ctx, span = nopTracer.Start(telemetry.ContextWithDisabledTracing(ctx), SpanName)
```

Comment:
Okrem pouzitia `nopTracer` (`nopSpan` - ktory je ale private a neda sa samostatne vytvorit)
musime poznacit do `ctx`, ze v dalsom kode odvodenom od daneho `ctx` nechceme generovat spans.

Je to preto, lebo tracer berie `nopSpan` ako keby ani neexistovat a pri najblizsom spane o uroven nizsie by vytvoril root span.